build.gradle

```diff
@@ -194,12 +196,14 @@ project(":core"){
     apply plugin: "java"
 
     task preGen{
-        outputs.upToDateWhen{ false } 
+        outputs.upToDateWhen{ false }
         generateLocales()
         writeVersion()
     }
 
     dependencies{
+		// Set corePath to current project folder
+		System.properties["corePath"] = file(Paths.get(".")).toString()
```

Comment:
Why not use ``file("").absolutePath`, or something similar? Importing `nio` doesn't seem necessary.

Reply:
Fixed.